src/OpenConext/ProfileBundle/Resources/views/MyProfile/overview.html.twig

```diff
@@ -6,8 +6,34 @@
 
 {% block content %}
     <h1>{{ 'profile.my_profile.long_title'|trans }}</h1>
-    <div class="table-overview-container">
-        <table class="overview-table mdl-data-table">
+    <p>{{ 'profile.my_profile.introduction'|trans }}</p>
+    <p>{{ 'profile.my_profile.explanation'|trans }}</p>
+    <p>{{ 'profile.my_profile.pi_processing'|trans }}</p>
+
+    {% if user.hasSupportContactEmail %}
+        {% set supportContactEmail = user.supportContactEmail|replace({'mailto:': ''}) %}
+
+    <p>{{ 'profile.my_profile.questions'|trans }} <a href="mailto: {{ supportContactEmail }}"><i class="prepended fa fa-envelope"></i>{{ supportContactEmail }}</a>.</p>
```

Comment:
Doesn't the space after `mailto:` break things? To be sure, I'd rather remove it :p